src/client/client.cpp

```diff
@@ -353,11 +352,6 @@ Client::~Client()
 
 	guiScalingCacheClear();
 
-	delete m_minimap;
-	m_minimap = nullptr;
-
-	delete m_media_downloader;
```

Comment:
Would be safer to use explicit `reset()` calls here, to preserve destruction order.

Reply:
I agree, but I don't trust our code.